### PR TITLE
Update template for FreeMcServer.net

### PR DESCRIPTION
**Pull request by Levii22**

### CODE REVIEW

File: _data/providers.json [hunk 1]
```diff
@@ -185,7 +185,7 @@
     {
       "name": "FreeMcServer.net",
       "url": "https://freemcserver.net",
-      "description": "Get Geyser as a plugin. Create a UDP port in the ports tab then type it in your config and connect with that port."
+      "description": "default"
```
Comment (Chew) [inline _data/providers.json:188]:
```suggestion
      "description_template": "default"
```

**Review by Chew — APPROVED** [review]

**Comment by Camotoy** [comment]
Is this a beneficial change? Seems less specific than the default.

**Comment by Levii22** [comment]
> 

Yes, we now allow this way of using geyser by using same port as the java server so no need to create special ports anymore.

**Comment by Camotoy** [comment]
Thanks, and thanks for waiting.

**Pull request merged**